server/meca/file-generators/article.test.js

```diff
@@ -1,8 +1,8 @@
 const generateXml = require('./article')
-const replaySetup = require('../../test/helpers/replay-setup')
+const Replay = require('replay')
 const sampleManuscript = require('./article.test.data')
 
-replaySetup('success')
+Replay.fixtures = `${__dirname}/../test/http-mocks`
```

Comment:
What was the reason for this?

Reply:
For the change? Trying to keep the meca package more self contained. Easier to keep track of the mock responses if they're local rather than one folder for the whole project.